docs/pages/router/reference/sitemap.mdx

```diff
@@ -27,7 +27,24 @@ You can also search for links directly in a browser like Safari or Chrome to tes
 
 Expo Router currently injects a **/\_sitemap** automatically that provides a list of all routes in the app. This is useful for debugging.
 
-To remove it, add a **app/\_sitemap.tsx** and return a null component:
+In SDK 52 and above, the sitemap can be removed by adding `sitemap: false` to the `expo-router` config plugin in the app config:
+
+```json app.json
+{
+  "plugins": [
+    [
+      "expo-router",
+      {
+        /* @info Disable the sitemap generation. */
+        "sitemap": false
+        /* @end */
+      }
+    ]
+  ]
+}
+```
+
+In SDK 51 and below, the sitemap can be removed by creating an empty `_sitemap` file.
```

Comment:
```suggestion
In SDK 51 and below, the sitemap can be removed by creating an empty **_sitemap.tsx** file inside **app** directory.
```

Let's make file names bold as per our writing style guide and also mention that create this file inside app directory.